package.nls.json

```diff
@@ -394,6 +394,8 @@
     "DataScience.insertAbove": "Insert cell above",
     "DataScience.addCell": "Add cell",
     "DataScience.runAll": "Run all cells",
+    "DataScience.runAbove": "Run cells above",
```

Comment:
We already have some command titles from the old code. Do any of the old strings work here so that we don't need to add new ones?

Reply:
We never had, the ones on toolbar.tsx for example were returning a default.

There are messages for the command palette:
"jupyter.command.jupyter.runallcellsabove.palette.title": "Run Cells Above Current Cell",
"jupyter.command.jupyter.runcurrentcellandallbelow.palette.title": "Run Current Cell and Below",

Should I use those instead? They're a bit different

Reply:
Hmmm, maybe just leave as is then. Keeps the native notebook stuff separate.